lib/output.js

```diff
@@ -597,6 +598,12 @@ function OutputStream(options) {
             return true;
         }
 
+        if (output.option('webkit')
+            && output.parent(0) instanceof AST_PropAccess
+            && output.parent(1) instanceof AST_Assign) {
```

Comment:
... except that now I'm getting failure from `pdfjs.js` in `test/jetstream.js` - investigating.

Command line:
```
node --turbo test/jetstream.js -mc keep_fargs=false,passes=3,pure_getters,unsafe,unsafe_comps,unsafe_math,unsafe_proto -b beautify=false,webkit
```